lib/cogctl/actions/relays/delete.ex

```diff
@@ -14,20 +14,28 @@ defmodule Cogctl.Actions.Relays.Delete do
   end
 
   defp do_delete(endpoint, relay_names) do
-    Enum.reduce(relay_names, 0, fn(relay_name, acc) ->
+    results = Enum.reduce(relay_names, %{success: [], failure: []}, fn(relay_name, acc) ->
```

Comment:
@christophermaier  how would that work. Wouldn't I need to iterate twice? Or am I missing something?